games-roguelike/FTL-gog/FTL-gog-1.6.7.ebuild

```diff
@@ -7,7 +7,7 @@ inherit desktop unpacker
 
 DESCRIPTION="Top-down roguelike space ship simulator"
 HOMEPAGE="https://www.gog.com/game/faster_than_light"
-SRC_URI="ftl_advanced_edition_en_1_6_3_17917.sh"
+SRC_URI="ftl_advanced_edition_en_${PV//./_}_18662.sh"
```

Comment:
Well, I meant including `.18662` in the ebuild filename as well. Any reason not to?

Reply:
The version number is only published as "1.6.7". We would be making an assumption about what is the value 18662. If you want me to consume this value anyway into the version number then I'd be renaming the ebuild to 1.6.7.18662?

Reply:
That shouldn't harm. You can go for sth like `1.6.7_p18662` if you really distrust it. But it's gonna be harder to process.